team1_pii.py

```diff
@@ -5,37 +5,66 @@
 # https://developers.google.com/edu/python/regular-expressions
 # https://regex101.com/
 def find_us_phone_numbers(text):
-    match = re.search(r'(\d{3}[-.]\d{3}[-.]\d{4})', text)
+    match = re.search(r'(\(?\d{3}\)?(\s{1}|[-.])\d{3}[-.]\d{4})', text)
     if match:
         return True
     return False
 
+
 def find_us_street_address(text):
-    match = re.search(r'(\d{1,4}\s{1}[a-zA-z\s]*,\s{1}[a-zA-z\s-]*,\s{1}[A-Z]{2},\s{1}(\d{5}-\d{4}|\d{5}))', text)
-    if match:
+    matchLong = re.search(r'(\d{3,4}\s{1}[a-zA-z\s]*,\s{1}[a-zA-z\s-]*,\s{1}[A-Z]{2},\s{1}(\d{5}-\d{4}|\d{5}))', text)
+    matchShort = re.search(
+        r'\d{3,4}\s{1}[a-zA-Z]*\s((\d{1,3}[a-zA-Z]*\s?)*|[a-zA-Z\s]*)(Street|Terrace|Lane|Lanes|Drive|Alley|Circle|Avenue|Boulevard){1}',
+        text)
+
+    if matchLong or matchShort:
         return True
     return False
 
+
 def find_credit_card_number(text):
     match = re.search(r'\d{4}[- ]\d{4}[- ]\d{4}[- ]\d{4}', text)
     if match:
         return True
     return False
 
+
 def find_twitter_handle(text):
     match = re.search(r'\B@\w*([A-Za-z0-9_]+)\w*', text)
     if match:
         return True
     return False
 
+
 def find_bank_acc_number(text):
-    match = re.search(r'^(\d{10}|\d{12})$', text)
+    match = re.search(r'(?:\d{3,4}\-\d{5}|\d{10}|\d{12})', text)
     if match:
         return True
     return False
 
+
 def find_email_address(text):
-    match = re.search(r'^([\w!-#+$&~?*=]+@[\w.]+)$', text)
+    match = re.search(r'[a-zA-Z0-9+_.-]+@[a-zA-Z0-9.-]+$', text)
```

Comment:
When you changed your regex you caused an existing test case to fail
```
        # Invalid Test
        email = '..nc12@email.com'
        self.assertFalse(find_email_address(email))
```

Reply:
I'll point out that because you are ending your regex with $ that a test case such as this would not pass
```
        email = 'my emails is ro1s3@email.com but what is yours?'
        self.assertTrue(find_email_address(email))
```
However, you got lucky and all the email in the test files does happen to be at the end of a line so no need to change for this assignment.